### PR TITLE
Fixed vitals validation bug, modified tests and readme instructions to run on web18

### PR DESCRIPTION
**Pull request by shinair**
Made the following changes:
1. Fixed the vitals validation logic - added the code in vitals.js instead of edit.js to ensure validation only happens on the vitals page form.
2. Modified the setup/teardown code for Selenium tests for compatibility with the Ubuntu server.
3. Modified the readme file for Selenium tests and added installation instructions along with the command to run the tests on the Ubuntu server.

### CODE REVIEW

File: interface/forms/vitals/vitals.js [hunk 1]
```diff
@@ -57,12 +57,18 @@
         let value = node.value;
         let inputSave = document.getElementById(targetSaveUnit);
         if (!inputSave) {
-            console.error("Failed to find node with data-target-input of ", targetSaveUnit);
+            console.error(
```
Comment (sunbiz) [inline interface/forms/vitals/vitals.js:60]: Can we remove the style changes to the vitals.js file.

File: interface/forms/vitals/vitals.js [hunk 2]
```diff
@@ -57,12 +57,18 @@
...
+                "Failed to find node with data-target-input of ",
+                targetSaveUnit
+            );
             return;
         }
         let inputConv = document.getElementById(targetInputConv);
         if (!inputConv) {
-            console.error("Failed to find node with data-target-input-conv of ", targetInputConv);
+            console.error(
```
Comment (sunbiz) [inline interface/forms/vitals/vitals.js:68]: Can we remove the style changes, same as above.

File: interface/forms/vitals/vitals.js [hunk 3]
```diff
@@ -77,27 +83,65 @@
                     inputSave.value = value;
                 }
             } else {
-                console.error("Failed to get valid number for input with id ", node.id, " with value ", value);
+                console.error(
```
Comment (sunbiz) [inline interface/forms/vitals/vitals.js:86]: Can we remove the style changes, same as above.

File: interface/forms/vitals/vitals.js [hunk 4]
```diff
@@ -77,27 +83,65 @@
...
+                    value
+                );
             }
         } else {
             inputSave.value = "";
             inputConv.value = "";
         }
 
-        if (targetSaveUnit == "weight_input" || targetSaveUnit == "height_input") {
+        if (
```
Comment (sunbiz) [inline interface/forms/vitals/vitals.js:98]: Can we remove the style changes, same as above.

File: interface/forms/vitals/vitals.js [hunk 5]
```diff
@@ -77,27 +83,65 @@
...
 
     function initDOMEvents() {
-        let vitalsForm = document.getElementById('vitalsForm');
+        let vitalsForm = document.getElementById("vitalsForm");
         if (!vitalsForm) {
             console.error("Failed to find vitalsForm DOM Node");
             return;
         }
-        vitalsForm.addEventListener('submit', vitalsFormSubmitted);
+        vitalsForm.addEventListener("submit", vitalsFormSubmitted);
```
Comment (sunbiz) [inline interface/forms/vitals/vitals.js:112]: Can we remove the style changes, same as above.

File: oemr-selenium/cdr_rule.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        actions.move_to_element(adminMenu).perform()
+
+        practiceMenu = self.browser.find_element(By.XPATH,
+                                                 '//*[@id="mainMenu"]/div/div[10]/div/ul/li[4]/div/div')
+
+        actions.move_to_element(practiceMenu).perform()
+
+        rules = self.browser.find_element(By.XPATH, '//*[@id="mainMenu"]/div/div[10]/div/ul/li[4]/div/ul/li[2]/div')
+        rules.click()
+
```
Comment (sunbiz) [inline oemr-selenium/cdr_rule.py:50]: Please remove blank lines.

File: interface/forms/vitals/vitals.js (hunk 1 above)
Comment (shinair) [reply interface/forms/vitals/vitals.js]: Removed the styling/formatting changes.

File: interface/forms/vitals/vitals.js (hunk 2 above)
Comment (shinair) [reply interface/forms/vitals/vitals.js]: Removed the styling/formatting changes.

File: interface/forms/vitals/vitals.js (hunk 3 above)
Comment (shinair) [reply interface/forms/vitals/vitals.js]: Removed the styling/formatting changes.

File: interface/forms/vitals/vitals.js (hunk 4 above)
Comment (shinair) [reply interface/forms/vitals/vitals.js]: Removed the styling/formatting changes.

File: interface/forms/vitals/vitals.js (hunk 5 above)
Comment (shinair) [reply interface/forms/vitals/vitals.js]: Removed the styling/formatting changes.

File: oemr-selenium/cdr_rule.py (hunk 6 above)
Comment (shinair) [reply oemr-selenium/cdr_rule.py]: Removed the blank lines.

**Pull request merged**